### PR TITLE
AP-72-JoobleSearchObject

### PR DESCRIPTION
**Pull request by FreshClouds**
Anyone else not sure where the start of AP-69 is to merge this into??

think I picked the right one...

Jooble Search Criteria Object that makes a Cron Job for each search criteria running daily.

### CODE REVIEW

**Comment by fritzb100** [comment]
You're right after this merge we need to reconcile and add AP-69 to dev

File: force-app/main/default/classes/JobSearchTriggerHandler_Test.cls [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            Assert.isTrue(e.getMessage().contains('DUPLICATE_VALUE, duplicate value found:'), 'Expected failure duplicate value found.');
+        }
+    }
+    @TestSetup
+    static void makeData(){
+        JobSearchCriteria__c defaultSearch = new JobSearchCriteria__c(
+            IsActive__c = true,
+            StatePick__c = 'Maryland',
+            City__c = 'Accident',
+            JobTitle__c = 'Digital Overlord'
```
Comment (fritzb100) [inline force-app/main/default/classes/JobSearchTriggerHandler_Test.cls:85]: Yes, now that's a job title one can be proud of :)

File: force-app/main/default/classes/JobSearchTriggerHandler_Test.cls [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        Test.stopTest();
+        List<CronTrigger> cronList = [SELECT CronJobDetail.Name, NextFireTime, Id FROM CronTrigger WHERE CronJobDetail.Name = 'Digital Overlord Accident, Maryland'];
+        Assert.areEqual(0, cronList.size());
+    }
+    @isTest
+    static void testForcedLimit(){
+        List<JobSearchCriteria__c> jobSearchList = new List<JobSearchCriteria__c>();
+        for(Integer i = 0 ; i < 51; i++ ){
+            JobSearchCriteria__c bulkSearchInsert = new JobSearchCriteria__c(
+            IsActive__c = true,
```
Comment (fritzb100) [inline force-app/main/default/classes/JobSearchTriggerHandler_Test.cls:33]: Is there a test for inserting 50 and it working?

**Review by fritzb100 — APPROVED** [review]
Only one question in the test class about when trying to insert 50 cron jobs

**Review by FreshClouds — COMMENTED** [review]
> Is there a test for inserting 50 and it working? 

Let me add one real quick!

File: force-app/main/default/classes/JobSearchTriggerHandler_Test.cls (hunk 2 above)
Comment (FreshClouds) [reply force-app/main/default/classes/JobSearchTriggerHandler_Test.cls:33]: bulk test for 50 now in just for fun as testAtLimit

**Pull request merged**